static/docs/command-reference/get.md

```diff
@@ -1,7 +1,8 @@
 # get
 
-Download or copy file or directory from any <abbr>DVC project</abbr> in a Git
-repository (e.g. hosted on GitHub) into the current working directory.
+Download or copy file or directory from the
+[remote storage](/doc/command-reference/remote) of any <abbr>DVC project</abbr>
```

Comment:
it will be not only from the remote project, we will need to fix it as a soon the ticket to is ready in the DVC core project.

Reply:
OK will keep in mind. But right now this is correct?

Reply:
yes, this is correct right now